package.json

```diff
@@ -4,6 +4,7 @@
   "repository": "https://github.com/poanetwork/cryptokitties-xdai-demo.git",
   "license": "GPL-3.0-only",
   "scripts": {
+    "initialize": "git submodule update --init && yarn install",
```

Comment:
Will `yarn install` initialize dependencies for the submodule as well? Is it required to resolve dependencies for the submodule - we are picking up only contract sources from there?

Reply:
`yarn install` will not install dependencies for submodule. We are using only the contracts sources from the submodule so dependencies are not needed

Reply:
So, if we suggest to init the submodule as part of the very first step (when cloning the repo) should we still have `git submodule update --init` here?

Reply:
Updated instructions, now that submodule is initialized as part of the first step, then we only need to install dependencies https://github.com/poanetwork/cryptokitties-xdai-demo/pull/6/commits/c62e63423f168b87f8bb15d16644ce5ee6485003

Reply:
thanks